### DISCUSSION TITLE
@furtif

### DISCUSSION THREAD

**Discussion by AndyIreland** [cedricp/ddt4all#833] [General]
@Furtif - could you stop with the 'closed' issues from things going back to 2018 I am getting massive amounts of email notifications!

**Comment by shrlnm** [comment]
@AndyIreland I suppose @Furtif do a good job. He close "issue" that are "not issue".   500+ issues on this project is not because the project is bad but vice verse because  a project very popular

**Reply by Furtif** [reply]
yep go make a big clean

**Reply by Furtif** [reply]
if @cedricp doesn't scold me....

**Comment by Furtif** [comment]
@cedricp as make me as manager and i manage .... as several issues is not realy issue but as question uses this https://github.com/cedricp/ddt4all/discussions and not make more issues code is ok issue is side code and not question

**Comment by Furtif** [comment]
all recent issues will be preserved, since you confuse 'issue' with a real issue an issue on github concerns the code error in code not itself all issues in question.

**Comment by Furtif** [comment]
everything that is a question given as an issue will be deleted and the link to discussions will be given here, if the code has an error issue in code it will be taken into account and seen seriously , now issues will be here ... , it will be normal to have 50k of notifications during the process I don't like to leave things half done

**Comment by Furtif** [comment]
after that everything will be back to normal and 3 issues in real...

**Comment by Furtif** [comment]
I apologize for the damage but it has to be done

**Comment by cedricp** [comment]
It seems it is possible to move an issue to the discussion board, BTW

**Reply by Furtif** [reply]
all part is not issue is a question

**Reply by Furtif** [reply]
lol
there is a question, they don't solve it, they put it in an issue .... usual

**Reply by Furtif** [reply]
close some and sen link of "discoutions" here

**Reply by Furtif** [reply]
I'm going to say that I'm furious to close everything

**Reply by Furtif** [reply]
and I happen to be ... if something doesn't have an error, I don't see why so much issue

**Comment by Furtif** [comment]
i thinks discution as @furtif not need i n this delete ... i prefered cedric is you repo i have only send ideas

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/835#discussion-5258595

**Comment by Furtif** [comment]
instead of walking there what where how ... answer the staff if you know

**Comment by Furtif** [comment]
other than that the cleaning continues

**Comment by Furtif** [comment]
it would be simpler if the Onwer of git (@cedricp ) went to settings closed issues then reopens it would close 500 ....... but good there is always a message of support etc and it closes one by one....

I'm not old, I just like things right ... if the code is correct the issues in question mode are making this repos rotten

**Comment by Furtif** [comment]
in any case, let's be respectful... in each case that I close, I leave a justification

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/113#issuecomment-1574017242

**Comment by Furtif** [comment]
sorry to be so direct

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/781#issuecomment-1574031478

I seem to be furious but.... I'm sorry.... many complain with everything under their noses

**Comment by Furtif** [comment]
I'm going to stop for a bit if they won't say that I'm a depraved madman.... but I'll remove what is not an issue from the repo.. I'll do that

**Comment by Furtif** [comment]
I'm going to put this clean repo and if you have a complaint or change of idea without being related to the code... it's right here, not in issues

**Comment by Furtif** [comment]
put yourselves at ease, I'm not one for half measures

**Comment by Furtif** [comment]
![image](https://github.com/cedricp/ddt4all/assets/11375035/cd4a3e75-84ad-4a06-b8aa-669a81fc0864)

**Comment by Furtif** [comment]
there are so many more or less...

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/580

**Comment by Furtif** [comment]
pffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff

**Comment by Furtif** [comment]
i close all ..............

**Comment by Furtif** [comment]
end.. https://github.com/cedricp/ddt4all/discussions/839

**Comment by cedricp** [comment]
Good job 👍

**Comment by Furtif** [comment]
this is not more util! go close
